Assets/00_Content/Scripts/Interactables/PickUp.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace Interactables {
+
+	public class PickUp : MonoBehaviour, IPickUp {
+
+		[SerializeField]
+		Transform itemGrabTransform;
+
+		public void DropItem() {
+
+			if (itemGrabTransform == null) {
+				transform.SetParent(null);
+				return;
+			}
+
+			itemGrabTransform.SetParent(null);
+		}
+
+		public void PickUpItem(Transform playerGrabTransform) {
+
+			//TODO - Move to players hands
+
+			if (itemGrabTransform == null) {
+				Debug.Log("No grabPoint set for item, using default position");
+				transform.SetParent(playerGrabTransform);
+				return;
+			}
+
+			itemGrabTransform.SetParent(playerGrabTransform);
```

Comment:
So this parents the items grab point (which is a child of the item itself) to the players "hand"? Wouldn't that just detach the grab point and leave the item on the ground?

Reply:
good point lol nice catch

Reply:
test